### PR TITLE
[bitnami/mongodb] Revisit tests

### PR DESCRIPTION
**Pull request by joancafom**
### Description of the change

After reviewing the existing tests and the general testing strategy we have been following until now, we saw there was a need to revisit them. The main objective that our chart testing strategy should aim for is asserting that the Chart deliverable works. As such, the focus should not be on the application itself or the container, which are regarded as trustful components (that is, they should have been tested at a previous stage).

As charts usually pack a number of different components, it is essential to test that their integrations and the application itself work as expected.

Consequently, we have updated the tests so they are related to the chart deliverable and left only a couple of basic tests to verify that the application functions correctly. Besides that, an effort has been made to reduce future tests' flakiness improving some existing selectors and checks.

Tested in https://github.com/joancafom/charts/pull/79

### Benefits

Improved, revisited test strategy.

### Possible drawbacks

None

### Applicable issues

NA

### Additional information

NA

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by agomezmoron — APPROVED** [review]

File: .vib/mongodb/goss/goss.yaml [hunk 1]
```diff
@@ -1,33 +1,36 @@
-process:
-  mongod:
-    running: true
+file:
+  /bitnami/mongodb:
```
Comment (FraPazGal) [inline .vib/mongodb/goss/goss.yaml:2]: I think we should add `persistence.mountPath` to `runtime_params` and the vars file and parametrize this.

File: .vib/mongodb/goss/goss.yaml [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+  {{- $conn_opts := printf "--quiet --username %s --password '%s' --host mongodb --port %d" .Vars.auth.rootUser .Vars.auth.rootPassword .Vars.service.ports.mongodb }}
   admin-command:
-    exec: mongosh --quiet --username "$MONGODB_ROOT_USER" --password "$MONGODB_ROOT_PASSWORD" --port "$MONGODB_PORT_NUMBER" --eval "db.adminCommand({listCollections:1})"
+    exec: mongosh {{ $conn_opts }} --eval "db.adminCommand({listCollections:1})"
     exit-status: 0
-    stdout:
-      - "system.version"
-    stderr: [ ]
-    timeout: 20000
+    timeout: 5000
```
Comment (FraPazGal) [inline .vib/mongodb/goss/goss.yaml:22]: IIRC, the default timeout is 10s so maybe we can remove this line if reducing it is not a must. Same to with the test below.

**Review by FraPazGal — COMMENTED** [review]
Hi @joancafom, the PR looks good overall! I left a couple of comments, could you please take a look?

File: .vib/mongodb/goss/goss.yaml (hunk 2 above)
Comment (joancafom) [reply .vib/mongodb/goss/goss.yaml:22]: Yeah, thanks. Don't really know why I added it tbh.

File: .vib/mongodb/goss/goss.yaml (hunk 1 above)
Comment (joancafom) [reply .vib/mongodb/goss/goss.yaml]: Done, thanks.

**Review by agomezmoron — APPROVED** [review]

**Review by FraPazGal — APPROVED** [review]

**Pull request merged**